scripts/ansible

```diff
@@ -91,12 +92,11 @@ elif [ -f /etc/debian_version ]; then    # Includes Debian, Ubuntu & Raspbian
     #wget http://download.iiab.io/packages/ansible_2.4.2.0-1ppa~xenial_all.deb
     #apt -y --allow-downgrades install ./ansible_2.4.2.0-1ppa~xenial_all.deb
 
-    echo -e 'PPA source "deb http://ppa.launchpad.net/ansible/ansible/ubuntu bionic main"'
-    echo -e "successfully saved to /etc/apt/sources.list.d/iiab-ansible.list\n"
-
-    echo -e "IF *OTHER* ANSIBLE SOURCES APPEAR BELOW, PLEASE MANUALLY REMOVE THEM TO"
-    echo -e 'ENSURE ANSIBLE UPDATES CLEANLY: (then re-run this script to be sure!)\n'
-    grep '^deb .*ansible' /etc/apt/sources.list /etc/apt/sources.list.d/*.list | grep -v '^/etc/apt/sources.list.d/iiab-ansible.list:' || true    # Override bash -e (instead of aborting at 1st error)
+#    echo -e 'PPA source "deb http://ppa.launchpad.net/ansible/ansible/ubuntu bionic main"'
```

Comment:
Line 95-99 help people who've ended up putting too many different Andible sources into /etc/apt/sources.list.d/*.list — giving them important guidance on how to get back on track.

So can we move these 5 lines, so they live just prior to the `fi` on Line 77 of this PR?

Reply:
ok prior to 77 works, was looking to limit misleading information.